docs/dev/stream/operators/windows.md

```diff
@@ -730,9 +730,9 @@ input
 
 /* ... */
 
-public class MyProcessWindowFunction implements ProcessWindowFunction<Tuple<String, Long>, String, String, TimeWindow> {
+public static class MyProcessWindowFunction extends ProcessWindowFunction<Tuple<String, Long>, String, String, TimeWindow> {
```

Comment:
In other code samples, we don't put the `static`, not assuming that this is defined as an inner class

Reply:
make sense! 
I see some `static` in this docs for other examples, we might need to make them consistent

`private static class MyReduceFunction implements ReduceFunction<SensorReading> {`